doc/docstyle.rst

```diff
@@ -8,6 +8,8 @@ General Guidelines
   procs can be useful too (visible via ``nim doc --docInternal foo.nim``).
 * Within documentation, a period (`.`) should follow each sentence (or sentence fragment) in a comment block. The documentation may be limited to one sentence fragment, but if multiple sentences are within the documentation, each sentence after the first should be complete and in present tense.
 * Documentation is parsed as a custom ReStructuredText (RST) with partial markdown support.
+* Prefer single backtick to double backtick since it's simpler and nim doc supports it, even in rst files.
```

Comment:
-1. While we do support this in `nim doc`, RST is still RST and we do have users who use different doc generators. Besides, it's not that difficult to just type two backticks...

Reply:
I've changed the phrasing to: `In nim sources, prefer ...` so we can have that debate another day for rst files

And I'd like to know more about how many users actually use a different doc generator than `nim doc` or `nim rst2html`; is that a real or theoretical argument? in any case, because of the "custom rst/markdown" support, i doubt a standard rst parser would be much help anyways; plus, such users would miss out on important features such as generation of theindex + doc search which is now enabled by default and will get further improvements.

>  it's not that difficult to just type two backticks

it adds noise/friction and a large majority of users are familiar with markdown, not rst.